Formula/fheroes2.rb

```diff
@@ -43,6 +43,20 @@ def install
     bin.install "script/homm2/extract_homm2_resources.sh" => "fheroes2-extract-resources"
   end
 
+  def caveats
+    <<~EOS
+      Please visit project website:
+
+      > #{homepage}
+
+      and read
+
+      > #{prefix}/share/doc/fheroes2/README.txt
+
+      for full installation instructions and more information.
```

Comment:
People already rarely read caveats so this won't help. I've reduced it to the only part that is specific to Homebrew.
```suggestion
      Documentation is available at:
      #{prefix}/share/doc/fheroes2/README.txt
```